Classes/RedirectStorage.php

```diff
@@ -289,8 +291,10 @@ protected function updateDependingRedirects(RedirectInterface $newRedirect): arr
                 } else {
                     $obsoleteRedirect->setTargetUriPath($newRedirect->getTargetUriPath());
                 }
+                $obsoleteRedirect->setStatusCode($newRedirect->getStatusCode());
```

Comment:
this seems to be the _real_ change?

Reply:
yes https://github.com/neos/redirecthandler-databasestorage/pull/28#issuecomment-1628311122